blueocean-personalization/src/main/js/model/User.js

```diff
@@ -18,6 +19,10 @@ export class User extends Record(
     isAnonymous() {
         return false;
     }
+
+    static current() {
+        return new User(AppConfig.getInitialUser());
```

Comment:
Sorry to be annoying about this, I just wish we could get any code dealing with auth or user into the same lib (`Security`). Maybe this `getInitialUser` should be pseudo-private, or not exist at all (let `Security` reach in and grab the object)